unlock-app/src/hooks/useProvider.ts

```diff
@@ -203,7 +204,7 @@ export const useProvider = (config: any) => {
     setNetwork(undefined)
     setConnected(undefined)
 
-    clearStorage()
+    clearStorageWithoutFavorite()
```

Comment:
That's where you would pass `clearStorage(["x", "y",...])` where `x`, `y` etc are the keys to delete.